AndroidAnnotations/androidannotations-ormlite/ormlite-api/src/main/java/org/androidannotations/ormlite/api/OrmLiteInjectHelper.java

```diff
@@ -0,0 +1,8 @@
+package org.androidannotations.ormlite.api;
+
+/**
+ * @param <T> type to inject
+ */
+public interface OrmLiteInjectHelper<T> {
+	T inject();
+}
```

Comment:
currently that would be hard to implement as the handler itself only returns an `IJExpresson` that "returns" the desired object.